mnist_simple/Makefile

```
@@ -0,0 +1,33 @@
+# This is a simple Makefile used to build the example source code.
+# This example might requires some modifications in order to work correctly on
+# your system.
+# If you're not using the Armadillo wrapper, replace `armadillo` with linker commands
+# for the BLAS and LAPACK libraries that you are using.
+
+TARGET := mnist_simple
+SRC := mnist_simple.cpp
+LIBS_NAME := armadillo mlpack boost_serialization
+
+CXX := g++
+CXXFLAGS := -std=c++11 -Wall -Wextra
+LDFLAGS  := -fopenmp
+DBGFLAG := -g
+INCLFLAGS := -I . # Add header directories if there is any.
```

```suggestion
INCLFLAGS := -I . # Add header directories if there are any.
```
Maybe a better comment might be `Add header directories for any includes that aren't on the default compiler search path.`?  Up to you.

---

agreed, the second one is better